OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -36,23 +40,556 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus target="#53ra"/>
+                                    <hi rend="rubric">ማርያም፡</hi> እመ፡ አዶኒ። ማዕበለ፡ ጌጋይ፡ ጽኑዕ፡ እስመ፡ ለአሥጥሞ፡ በጽሐኒ። ሐመርየኒ፡ ማዕዶትየኒ፡ ሃይማኖትየኒ፡ 
+                                    ዘእንበሌኪ፡ አልብየ፡ ምንትኒ።
+                                </explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i1.29">
+                                <locus from="53ra" to="55va"/>
+                                <title type="complete" ref="LIT1864Malkea"/>
+                                <textLang mainLang="gez"/>
+                                <incipit xml:lang="gez">
+                                    <locus from="53ra" to="53rb"/>
+                                    <hi rend="rubric">ሰላም<supplied reason="omitted" resp="MV">፡</supplied></hi> ለሥዕልኪ፡ ዘይኤድም፡ ለንጻሬ። ዓዲ፡ ሰላም፡ ለሥዕለ፡ ወልድኪ፡ በኁባሬ።
+                                    አክሊለ፡ ሢመቱ፡ ለጴ<cb n="b"/>ጥሮስ፡ <hi rend="rubric">ማርያም፡</hi> እንተ፡ ትጸድሊ፡ እምወራውሬ። 
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus target="#55va"/>
+                                    ወእሰግድ፡ ሥልሰ፡ <hi rend="rubric">ማርያም፡</hi> ለሥዕልኪ፡ ቅድመ፡ ጉባዔ፡ ዘኢሰገደሰ፡ ለሥዕልኪ፡ አሥሪፆ፡ ምማዒ፡ በሥጋሁ፡ ወበነፍሱ፡ ኢይርከብ፡
+                                    ትንሣኤ። <hi rend="rubric">ስብሐት፡</hi> ወክብር፡ ወሰጊድ፡ ለመንግሥትኪ፡ ይደሉ። እስመ፡ ዓባልኪ፡ ለእግዚአብሔር፡ ዓባሉ፡ <hi rend="rubric">ማርያም፡</hi>
+                                    በረከትኪ፡ ምስሌነ፡ የሃሉ፡
+                                </explicit>
+                            </msItem>                            
+                        <msItem xml:id="ms_i1.30">
+                            <locus from="55vb" to="57va"/>
+                            <title type="complete" ref="LIT3064RepCh344"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <locus target="#55vb"/>
+                                <hi rend="rubric">እሴብሕ፡ ጸጋኪ፡ ኦምልእተ፡ ጸጋ፡</hi> ወእትጋነይ፡ ለኪ፡ ኦምልእተ፡ ውዳሴ፡ ተዘከርኩ፡ በሌሊት፡ ንጽሐ፡ ድንግልናኪ፡ ዘኢማሰነ፡ በወሊድ፡ 
+                                ወኢረስሐ፡ በሐሪስ፡ እስመ፡ ቅድስተ፡ ቅዱሳን፡ አንቲ።
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus from="57rb" to="57va"/>
+                                ስፍሒ፡ እዴኪ፡ ወባርክኒ፡ ለገብርኪ። በረከተ፡ ዚአኪ፡ የሀሉ፡ ወጸጋ፡ ወልድኪ፡ ላዕለ፡ <hi rend="rubric">ተክለ፡ ሃይም<add place="interlinear">ኖት፡</add></hi> 
+                                መስሕኪ፡ ወላዕለ፡ ኵልነ፡ አ<pb n="57v"/><cb n="a"/>ግብርትኪ፡ ወአዕማትኪ፡ ለዓለመ፡ ዓለም፡ አሜን።
+                            </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.31">
+                            <locus from="57va" to="85vb"/>
+                            <title type="incomplete" ref="LIT6289Semani"/>
```

Comment:
Maybe you could add a quick note on the nature of the incompleteness? THe end is missing?

Reply:
I've added the following remark: `<note>The text concludes abruptly. The end is apparently missing.</note>`